wagtail/contrib/forms/tests/test_views.py

```diff
@@ -361,6 +362,52 @@ def setUp(self):
         # Login
         self.login()
 
+    def get_url(self, viewname, *args, **kwargs):
+        """
+        Reverse a URL and append querystring arguments.
+        """
+        querystring = kwargs.pop("querystring", None)
+        url = reverse(viewname, args=args, kwargs=kwargs)
+        if querystring:
+            url += "?" + urlencode(querystring)
+        return url
+
+    def test_export_urls_include_filters(self):
+        # Set up some test submissions
+        submission = FormSubmission(page=self.form_page, form_data={"name": "Alice"})
+        submission.save()
+
+        # Apply some filters to the submissions view
+        self.client.get(
+            self.get_url(
+                viewname="wagtailforms:list_submissions",
+                page_id=self.form_page.id,
+                querystring={"name": "Alice"},
+            )
+        )
+
+        # Ensure that the xlsx_export_url and csv_export_url include the filter parameters
+        response = self.client.get(
+            self.get_url(
+                viewname="wagtailforms:list_submissions",
+                page_id=self.form_page.id,
+                querystring={"name": "Alice"},
+            )
+        )
+        view = response.context["view"]
+        expected_url = self.get_url(
+            viewname="wagtailforms:list_submissions",
+            page_id=self.form_page.id,
+            querystring={"name": "Alice", "export": "xlsx"},
+        )
+        self.assertIn(expected_url, response.content.decode())
+        expected_url = self.get_url(
+            viewname="wagtailforms:list_submissions",
+            page_id=self.form_page.id,
+            querystring={"name": "Alice", "export": "csv"},
+        )
+        self.assertIn(expected_url, response.content.decode())
```

Comment:
If you run this test, it will fail because the URLs will be HTML-escaped within the template. We will need to `escape` it before asserting.